### PR TITLE
Ncwms - parent onblur method called on blur event of datepicker

### PR DESCRIPTION
**Pull request by pmbohm**
Replaces my previous attempt to check on every character change in the datepicker. @jonescc

### CODE REVIEW

**Comment by dnahodil** [comment]
Sorry @pmbohm, this doesn't fix #1581 for me.

**Comment by pmbohm** [comment]
Yes ok the values can still be read by the download panel before they are loaded. To test this here's some code
DownloadPanel.js
```

    generateContent: function() {
        console.log("generate content");
        this.downloadPanelBody.generateBodyContent();
    }
```
NcWMS.js
```
    _timeSeriesLoadedForDate: function() {
        var that = this;
        setTimeout(function() { 
            console.log("_timeSeriesLoadedForDate");
            if (0 == that.pendingRequests.size()) {
                that._initSubsetExtent();

                if (!that.time) {
                    that._initToMostRecentTime();
                    that.params['TIME'] = that._getTimeParameter(that.time);
                }

                that.events.triggerEvent('temporalextentloaded', that);
                Ext.MsgBus.publish(PORTAL_EVENTS.ACTIVE_GEONETWORK_RECORD_ADDED);
            }
        }, 5000);
    },
```

**Comment by pmbohm** [comment]
Ok this is it at version4 @dnahodil

**Comment by pmbohm** [comment]
@dnahodil travis is done

**Comment by pmbohm** [comment]
@dnahodil New pull request is up just now

File: web-app/js/portal/ui/openlayers/layer/NcWMS.js [hunk 1]
```diff
@@ -160,9 +160,7 @@ OpenLayers.Layer.NcWMS = OpenLayers.Class(OpenLayers.Layer.WMS, {
 
     loadTimeSeriesForDay: function(date) {
         if (this.getTimeSeriesForDay(date)) {
-            // Give UI precedence by using delayed function call (setTimeout)
-            var that = this;
-            setTimeout(function() { that._timeSeriesLoadedForDate(); }, 10);
+            this._timeSeriesLoadedForDate();
```
Comment (dnahodil) [inline web-app/js/portal/ui/openlayers/layer/NcWMS.js]: This change looks unrelated, to me. Have you rebased this branch off master recently?
